### DISCUSSION TITLE
Adding Polhemus Patriot to Plus toolkit

### DISCUSSION THREAD

**Discussion by anaghamg** [PlusToolkit/PlusLib#1036] [General]
Hello,  I would like to add a new device Polhemus Patriot to Plus but im having trouble writing vtkplusPolhemus.cxx . Has anyone used Patriot before?
Any help would be appreciated!

i found 2 previous posts on this but its a dead end. 
https://github.com/PlusToolkit/PlusLib/discussions/903
https://github.com/PlusToolkit/PlusLib/issues/512

**Comment by Sunderlandkyl** [comment]
I don't think anybody has used Patriot with Plus before.

Are you following the guide on this page: https://plustoolkit.github.io/devicecode?
Where are you having trouble?

If you find that you are unable to implement the device, you could use a framegrabber to capture the output of the ultrasound in Plus.
